lib/cdo/rack/deflater.rb

```diff
@@ -107,13 +107,13 @@ def close
     end
 
     class DeflateStream
-      DEFLATE_ARGS ||= [
+      DEFLATE_ARGS = [
```

Comment:
Is there any reason to use `||=` rather than `=` here? Also in `shared/middleware/helpers/dynamo_table.rb`.

Reply:
Not that I know of. It was added [here](https://github.com/code-dot-org/code-dot-org/commit/fe96683603b249594104ab3964ea571ef79efa15#diff-f18a35f3136dc1e9909ff2af60d3bea7L128). @wjordan ?

Reply:
I don't recall why this change was made, it's possible the class was being reloaded and this avoids a constant-redefinition warning or something. (Neither do I recall whether the change was necessary or not.)

Since this is backported + un-maintained code, I'm indifferent to this change for style purposes as long as I'm not responsible for fixing it if it breaks anything.

As for the `dynamo_table` code, looks like @Bjvanminnen last touched that so you can check with him on that one.

Reply:
Apparently I made this change ~2 years ago, so my memory is hazy.

From what I do remember, when I've made changes like this it was for the reasons Will said - in some context I was getting a warning about constant redefinition.

Reply:
Roger, then I'll plan on reverting the change from `||=` to `=` to proactively avoid a constant redefinition warning as part of this PR.